### PR TITLE
0.6.15 enhancements

### PR DESCRIPTION
**Pull request by blakeneyops**
Extended examples and updated formatting to HashiCorp standard.

### CODE REVIEW

File: examples/full_stack/main.tf [hunk 1]
```diff
@@ -22,14 +22,26 @@ module "vpc_dhcp" {
...
+  domain_name          = "${var.domain_name}"
+  name_servers         = "${var.name_servers}"
+  ntp_servers          = "${var.ntp_servers}"
   netbios_name_servers = "${var.netbios_name_servers}"
-  netbios_node_type = "${var.netbios_node_type}"
+  netbios_node_type    = "${var.netbios_node_type}"
+}
+
+## Configures ACLs
+resource "aws_network_acl" "acl" {
```
Comment (yuhunglin) [inline examples/full_stack/main.tf:36]: @blakeneyops just curious, how does VPC network ACLs differ from SGs? Anything worth highlighting in README notes?

**Comment by yuhunglin** [comment]
👍  LGTM

File: examples/full_stack/main.tf (hunk 1 above)
Comment (yuhunglin) [inline examples/full_stack/main.tf:36]: Ah, captured [here](http://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/VPC_Security.html#VPC_Security_Comparison)

Comment (yuhunglin) [inline examples/full_stack/main.tf:36]: @blakeneyops is it worth rolling the `aws_network_acl` resource into the `base` module?

Comment (blakeneyops) [inline examples/full_stack/main.tf:36]: @yuhunglin I thought about that, but you have to establish the subnet mappings when declaring the resource. This is the only layer where we have both the VPC ID and the all the subnet IDs.

**Pull request merged**